### ISSUE TITLE
Using Visualize DB with @Lob propertie

### ISSUE THREAD

**Issue by hmbrito** [jGauravGupta/JPAModeler#125]
Message : Exception Description: The type [class byte []] for the attribute [foto] on the entity class [class Carro] is not a valid type for a lob mapping. For a lob of type BLOB, the attribute must be defined as a java.sql.Blob, byte[], Byte[] or a Serializable type. For a lob of type CLOB, the attribute must be defined as a java.sql.Clob, char[], Character[] or String type.

StackTrace : 
Local Exception Stack: 
Exception [EclipseLink-7164] (Eclipse Persistence Services - 2.6.2.v20151217-774c696): org.eclipse.persistence.exceptions.ValidationException
Exception Description: The type [class byte []] for the attribute [foto] on the entity class [class Carro] is not a valid type for a lob mapping. For a lob of type BLOB, the attribute must be defined as a java.sql.Blob, byte[], Byte[] or a Serializable type. For a lob of type CLOB, the attribute must be defined as a java.sql.Clob, char[], Character[] or String type.
	at org.eclipse.persistence.exceptions.ValidationException.invalidTypeForLOBAttribute(ValidationException.java:1135)
	at org.eclipse.persistence.internal.jpa.metadata.converters.LobMetadata.process(LobMetadata.java:124)
	at org.eclipse.persistence.internal.jpa.metadata.accessors.mappings.MappingAccessor.processLob(MappingAccessor.java:1760)
	at org.eclipse.persistence.internal.jpa.metadata.accessors.mappings.BasicAccessor.processLob(BasicAccessor.java:524)
	at org.eclipse.persistence.internal.jpa.metadata.accessors.mappings.MappingAccessor.processMappingConverter(MappingAccessor.java:1824)
	at org.eclipse.persistence.internal.jpa.metadata.accessors.mappings.MappingAccessor.processMappingValueConverter(MappingAccessor.java:1849)
	at org.eclipse.persistence.internal.jpa.metadata.accessors.mappings.BasicAccessor.process(BasicAccessor.java:419)
	at org.netbeans.jpa.modeler.db.accessor.BasicSpecAccessor.process(BasicSpecAccessor.java:56)
	at org.eclipse.persistence.internal.jpa.metadata.MetadataDescriptor.processMappingAccessors(MetadataDescriptor.java:1544)
	at org.eclipse.persistence.internal.jpa.metadata.accessors.classes.ClassAccessor.processMappingAccessors(ClassAccessor.java:1648)
	at org.eclipse.persistence.internal.jpa.metadata.accessors.classes.EntityAccessor.processMappingAccessors(EntityAccessor.java:1240)
	at org.eclipse.persistence.internal.jpa.metadata.accessors.classes.EntityAccessor.process(EntityAccessor.java:699)
	at org.eclipse.persistence.internal.jpa.metadata.MetadataProject.processStage2(MetadataProject.java:1808)
	at org.eclipse.persistence.internal.jpa.metadata.MetadataProcessor.processORMMetadata(MetadataProcessor.java:573)
	at org.netbeans.db.modeler.persistence.internal.jpa.deployment.JPAMPersistenceUnitProcessor.processORMetadata(JPAMPersistenceUnitProcessor.java:49)
	at org.netbeans.db.modeler.specification.model.util.DBModelerUtil.createDBMapping(DBModelerUtil.java:255)
	at org.netbeans.db.modeler.specification.model.util.DBModelerUtil.loadModelerFileInternal(DBModelerUtil.java:193)
	at org.netbeans.db.modeler.specification.model.util.DBModelerUtil.loadModelerFile(DBModelerUtil.java:178)
	at org.netbeans.modeler.core.NBModelerUtil.loadModelerFile(NBModelerUtil.java:57)
	at org.netbeans.modeler.specification.model.file.action.ModelerFileActionListener.openModelerFile(ModelerFileActionListener.java:143)
	at org.netbeans.db.modeler.specification.model.file.action.DBViewerActionListener.init(DBViewerActionListener.java:58)
	at org.netbeans.jpa.modeler.specification.model.util.JPAModelerUtil.openDBViewer(JPAModelerUtil.java:1587)
	at org.netbeans.jpa.modeler.specification.model.scene.JPAModelerScene.lambda$getPopupMenuItemList$4(JPAModelerScene.java:389)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.AbstractButton.doClick(AbstractButton.java:376)
	at javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:833)
	at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:877)
	at java.awt.Component.processMouseEvent(Component.java:6533)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:159)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

ModelerFile : 
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<jpa:entity-mappings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:java="http://jcp.org/en/jsr/detail?id=270" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:jpa="http://java.sun.com/xml/ns/persistence/orm" v="1.5.4" thm="PNBColorScheme" sm="false" xs="false" id="_14606602456751" xsi:schemaLocation="http://java.sun.com/xml/ns/persistence/orm orm_2_1.xsd">
    <jpa:entity xre="false" compositePrimaryKeyClass="AcessorioPK" abs="false" class="Acessorio" v="true" minimized="false" rootElement="_14606602456751" id="_14606602456802">
        <jpa:attributes>
            <jpa:basic optional="true" attribute-type="String" v="true" name="descricao" id="_146066024568512">
                <jpa:bv/>
                <jpa:column name="descricao" unique="false" nullable="true" insertable="true" updatable="true" table="acessorio" length="255"/>
            </jpa:basic>
            <jpa:many-to-many own="true" collection-type="java.util.List" connected-entity-id="_14606602456853" connected-attribute-id="_146066024568522" v="true" name="carroCollection" id="_146066024568513">
                <jpa:bv/>
                <jpa:join-table name="carro_acessorio">
                    <jpa:join-column name="codigo_acessorio" unique="false" nullable="true" insertable="true" updatable="true"/>
                    <jpa:inverse-join-column name="codigo_carro" unique="false" nullable="true" insertable="true" updatable="true"/>
                </jpa:join-table>
            </jpa:many-to-many>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568511">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="acessorio"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="acessorio"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="CarroPK" abs="false" class="Carro" v="true" minimized="false" rootElement="_14606602456751" id="_14606602456853">
        <jpa:attributes>
            <jpa:basic optional="true" attribute-type="String" v="true" name="chassi" id="_146066024568515">
                <jpa:bv/>
                <jpa:column name="chassi" unique="false" nullable="true" insertable="true" updatable="true" table="carro" length="255"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="String" v="true" name="cor" id="_146066024568516">
                <jpa:bv/>
                <jpa:column name="cor" unique="false" nullable="true" insertable="true" updatable="true" table="carro" length="255"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="java.util.Date" v="true" name="dataCriacao" id="_146066024568517">
                <jpa:bv/>
                <jpa:temporal>TIMESTAMP</jpa:temporal>
                <jpa:column name="dataCriacao" unique="false" nullable="true" insertable="true" updatable="true" table="carro"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="java.util.Date" v="true" name="dataModificacao" id="_146066024568518">
                <jpa:bv/>
                <jpa:temporal>TIMESTAMP</jpa:temporal>
                <jpa:column name="dataModificacao" unique="false" nullable="true" insertable="true" updatable="true" table="carro"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="byte []" v="true" name="foto" id="_146066024568519">
                <jpa:bv/>
                <jpa:column name="foto" unique="false" nullable="true" insertable="true" updatable="true" table="carro"/>
                <jpa:lob/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="String" v="true" name="placa" id="_146066024568520">
                <jpa:bv/>
                <jpa:column name="placa" unique="false" nullable="true" insertable="true" updatable="true" table="carro" length="255"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="java.math.BigDecimal" v="true" name="valorDiaria" id="_146066024568521">
                <jpa:bv/>
                <jpa:column name="valorDiaria" unique="false" nullable="true" insertable="true" updatable="true" table="carro" precision="19" scale="2"/>
            </jpa:basic>
            <jpa:many-to-one primaryKey="false" connected-entity-id="_146066024568510" connected-attribute-id="_146066024568559" v="true" name="codigoModelo" id="_146066024568523">
                <jpa:bv/>
                <jpa:join-column name="codigo_modelo" unique="false" nullable="true" insertable="true" updatable="true"/>
            </jpa:many-to-one>
            <jpa:one-to-many own="false" collection-type="java.util.List" connected-entity-id="_14606602456856" connected-attribute-id="_146066024568544" v="true" name="aluguelCollection" id="_146066024568524">
                <jpa:bv/>
            </jpa:one-to-many>
            <jpa:many-to-many own="false" collection-type="java.util.List" connected-entity-id="_14606602456802" connected-attribute-id="_146066024568513" v="true" name="acessorioCollection" id="_146066024568522">
                <jpa:bv/>
            </jpa:many-to-many>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568514">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="carro"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="carro"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="PessoaPK" abs="false" class="Pessoa" v="true" minimized="false" rootElement="_14606602456751" id="_14606602456854">
        <jpa:attributes>
            <jpa:basic optional="false" attribute-type="String" v="true" name="tipoPessoa" id="_146066024568525">
                <jpa:bv/>
                <jpa:column name="TIPO_PESSOA" unique="false" nullable="false" insertable="true" updatable="true" table="pessoa" length="31"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="String" v="true" name="cpf" id="_146066024568527">
                <jpa:bv/>
                <jpa:column name="cpf" unique="false" nullable="true" insertable="true" updatable="true" table="pessoa" length="255"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="java.util.Date" v="true" name="dataNascimento" id="_146066024568528">
                <jpa:bv/>
                <jpa:temporal>DATE</jpa:temporal>
                <jpa:column name="data_nascimento" unique="false" nullable="true" insertable="true" updatable="true" table="pessoa"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="String" v="true" name="nome" id="_146066024568529">
                <jpa:bv/>
                <jpa:column name="nome" unique="false" nullable="true" insertable="true" updatable="true" table="pessoa" length="255"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="String" v="true" name="sexo" id="_146066024568530">
                <jpa:bv/>
                <jpa:column name="sexo" unique="false" nullable="true" insertable="true" updatable="true" table="pessoa" length="255"/>
            </jpa:basic>
            <jpa:one-to-one own="true" primaryKey="false" connected-entity-id="_14606602456857" connected-attribute-id="_146066024568550" v="true" name="motorista" id="_146066024568531">
                <jpa:bv/>
            </jpa:one-to-one>
            <jpa:one-to-one own="true" primaryKey="false" connected-entity-id="_14606602456859" connected-attribute-id="_146066024568556" v="true" name="funcionario" id="_146066024568532">
                <jpa:bv/>
            </jpa:one-to-one>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568526">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="pessoa"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="pessoa"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="ApoliceSeguroPK" abs="false" class="ApoliceSeguro" v="true" minimized="false" rootElement="_14606602456751" id="_14606602456855">
        <jpa:attributes>
            <jpa:basic optional="true" attribute-type="Boolean" v="true" name="protecaoCausasNaturais" id="_146066024568534">
                <jpa:bv/>
                <jpa:column name="protecaoCausasNaturais" unique="false" nullable="true" insertable="true" updatable="true" table="apolice_seguro"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="Boolean" v="true" name="protecaoRoubo" id="_146066024568535">
                <jpa:bv/>
                <jpa:column name="protecaoRoubo" unique="false" nullable="true" insertable="true" updatable="true" table="apolice_seguro"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="Boolean" v="true" name="protecaoTerceiro" id="_146066024568536">
                <jpa:bv/>
                <jpa:column name="protecaoTerceiro" unique="false" nullable="true" insertable="true" updatable="true" table="apolice_seguro"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="java.math.BigDecimal" v="true" name="valorFranquia" id="_146066024568537">
                <jpa:bv/>
                <jpa:column name="valorFranquia" unique="false" nullable="true" insertable="true" updatable="true" table="apolice_seguro" precision="19" scale="2"/>
            </jpa:basic>
            <jpa:one-to-many own="false" collection-type="java.util.List" connected-entity-id="_14606602456856" connected-attribute-id="_146066024568545" v="true" name="aluguelCollection" id="_146066024568538">
                <jpa:bv/>
            </jpa:one-to-many>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568533">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="apolice_seguro"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="apolice_seguro"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="AluguelPK" abs="false" class="Aluguel" v="true" minimized="false" rootElement="_14606602456751" id="_14606602456856">
        <jpa:attributes>
            <jpa:basic optional="true" attribute-type="java.util.Date" v="true" name="dataDevolucao" id="_146066024568540">
                <jpa:bv/>
                <jpa:temporal>TIMESTAMP</jpa:temporal>
                <jpa:column name="dataDevolucao" unique="false" nullable="true" insertable="true" updatable="true" table="aluguel"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="java.util.Date" v="true" name="dataEntrega" id="_146066024568541">
                <jpa:bv/>
                <jpa:temporal>TIMESTAMP</jpa:temporal>
                <jpa:column name="dataEntrega" unique="false" nullable="true" insertable="true" updatable="true" table="aluguel"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="java.util.Date" v="true" name="dataPedido" id="_146066024568542">
                <jpa:bv/>
                <jpa:temporal>DATE</jpa:temporal>
                <jpa:column name="dataPedido" unique="false" nullable="true" insertable="true" updatable="true" table="aluguel"/>
            </jpa:basic>
            <jpa:basic optional="true" attribute-type="java.math.BigDecimal" v="true" name="valorTotal" id="_146066024568543">
                <jpa:bv/>
                <jpa:column name="valorTotal" unique="false" nullable="true" insertable="true" updatable="true" table="aluguel" precision="19" scale="2"/>
            </jpa:basic>
            <jpa:many-to-one primaryKey="false" connected-entity-id="_14606602456853" connected-attribute-id="_146066024568524" v="true" name="codigoCarro" id="_146066024568544">
                <jpa:bv/>
                <jpa:join-column name="codigo_carro" unique="false" nullable="true" insertable="true" updatable="true"/>
            </jpa:many-to-one>
            <jpa:many-to-one primaryKey="false" connected-entity-id="_14606602456855" connected-attribute-id="_146066024568538" v="true" name="codigoApoliceSeguro" id="_146066024568545">
                <jpa:bv/>
                <jpa:join-column name="codigo_apolice_seguro" unique="false" nullable="true" insertable="true" updatable="true"/>
            </jpa:many-to-one>
            <jpa:many-to-one primaryKey="false" connected-entity-id="_14606602456857" connected-attribute-id="_146066024568549" v="true" name="codigoMotorista" id="_146066024568546">
                <jpa:bv/>
                <jpa:join-column name="codigo_motorista" unique="false" nullable="true" insertable="true" updatable="true"/>
            </jpa:many-to-one>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568539">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="aluguel"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="aluguel"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="MotoristaPK" abs="false" class="Motorista" v="true" minimized="false" rootElement="_14606602456751" id="_14606602456857">
        <jpa:attributes>
            <jpa:basic optional="true" attribute-type="String" v="true" name="numeroCNH" id="_146066024568547">
                <jpa:bv/>
                <jpa:column name="numeroCNH" unique="false" nullable="true" insertable="true" updatable="true" table="motorista" length="255"/>
            </jpa:basic>
            <jpa:one-to-many own="false" collection-type="java.util.List" connected-entity-id="_14606602456856" connected-attribute-id="_146066024568546" v="true" name="aluguelCollection" id="_146066024568549">
                <jpa:bv/>
            </jpa:one-to-many>
            <jpa:one-to-one optional="false" primaryKey="false" connected-entity-id="_14606602456854" connected-attribute-id="_146066024568531" v="true" name="pessoa" id="_146066024568550">
                <jpa:bv/>
                <jpa:join-column name="codigo" unique="false" nullable="true" insertable="false" updatable="false"/>
            </jpa:one-to-one>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568548">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="motorista"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="motorista"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="FabricantePK" abs="false" class="Fabricante" v="true" minimized="false" rootElement="_14606602456751" id="_14606602456858">
        <jpa:attributes>
            <jpa:basic optional="false" attribute-type="String" v="true" name="nome" id="_146066024568552">
                <jpa:bv/>
                <jpa:column name="nome" unique="false" nullable="false" insertable="true" updatable="true" table="fabricante" length="30"/>
            </jpa:basic>
            <jpa:one-to-many own="false" collection-type="java.util.List" connected-entity-id="_146066024568510" connected-attribute-id="_146066024568560" v="true" name="modelocarroCollection" id="_146066024568553">
                <jpa:bv/>
            </jpa:one-to-many>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568551">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="fabricante"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="fabricante"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="FuncionarioPK" abs="false" class="Funcionario" v="true" minimized="false" rootElement="_14606602456751" id="_14606602456859">
        <jpa:attributes>
            <jpa:basic optional="true" attribute-type="String" v="true" name="matricula" id="_146066024568554">
                <jpa:bv/>
                <jpa:column name="matricula" unique="false" nullable="true" insertable="true" updatable="true" table="funcionario" length="255"/>
            </jpa:basic>
            <jpa:one-to-one optional="false" primaryKey="false" connected-entity-id="_14606602456854" connected-attribute-id="_146066024568532" v="true" name="pessoa" id="_146066024568556">
                <jpa:bv/>
                <jpa:join-column name="codigo" unique="false" nullable="true" insertable="false" updatable="false"/>
            </jpa:one-to-one>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568555">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="funcionario"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="funcionario"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="ModelocarroPK" abs="false" class="Modelocarro" v="true" minimized="false" rootElement="_14606602456751" id="_146066024568510">
        <jpa:attributes>
            <jpa:basic optional="true" attribute-type="String" v="true" name="descricao" id="_146066024568558">
                <jpa:bv/>
                <jpa:column name="descricao" unique="false" nullable="true" insertable="true" updatable="true" table="modelocarro" length="255"/>
            </jpa:basic>
            <jpa:many-to-one primaryKey="false" connected-entity-id="_14606602456858" connected-attribute-id="_146066024568553" v="true" name="fabricanteCodigo" id="_146066024568560">
                <jpa:bv/>
                <jpa:join-column name="fabricante_codigo" unique="false" nullable="true" insertable="true" updatable="true"/>
            </jpa:many-to-one>
            <jpa:one-to-many own="false" collection-type="java.util.List" connected-entity-id="_14606602456853" connected-attribute-id="_146066024568523" v="true" name="carroCollection" id="_146066024568559">
                <jpa:bv/>
            </jpa:one-to-many>
            <jpa:id attribute-type="Long" v="true" name="codigo" id="_146066024568557">
                <jpa:bv/>
                <jpa:column name="codigo" unique="false" nullable="false" insertable="true" updatable="true" table="modelocarro"/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:table name="modelocarro"/>
    </jpa:entity>
    <jpa:c/>
    <jpa:diagram>
        <plane elementRef="_14606602456751">
            <shape elementRef="_14606602456802">
                <Bounds x="212.0" y="441.0" width="156.0" height="131.0"/>
            </shape>
            <shape elementRef="_14606602456853">
                <Bounds x="519.0" y="187.0" width="169.0" height="299.0"/>
            </shape>
            <shape elementRef="_14606602456854">
                <Bounds x="1302.0" y="187.0" width="143.0" height="236.0"/>
            </shape>
            <shape elementRef="_14606602456855">
                <Bounds x="1062.0" y="187.0" width="191.0" height="194.0"/>
            </shape>
            <shape elementRef="_14606602456856">
                <Bounds x="777.0" y="187.0" width="176.0" height="236.0"/>
            </shape>
            <shape elementRef="_14606602456857">
                <Bounds x="1098.0" y="431.0" width="155.0" height="152.0"/>
            </shape>
            <shape elementRef="_14606602456858">
                <Bounds x="30.0" y="187.0" width="184.0" height="131.0"/>
            </shape>
            <shape elementRef="_14606602456859">
                <Bounds x="1342.0" y="546.0" width="164.0" height="131.0"/>
            </shape>
            <shape elementRef="_146066024568510">
                <Bounds x="302.0" y="187.0" width="170.0" height="152.0"/>
            </shape>
        </plane>
    </jpa:diagram>
</jpa:entity-mappings>
```

**Comment by jGauravGupta** [comment]
This bug is already detected and closed in local branch , Fix will be
available in next release v3.1 .

Thanks and kind regards,
Gaurav Gupta
<https://twitter.com/intent/follow?original_referer=https%3A%2F%2Fabout.twitter.com%2Fresources%2Fbuttons&ref_src=twsrc%5Etfw&region=follow_link&screen_name=jpamodeler&tw_p=followbutton>
<http://jpamodeler.github.io/>   <https://www.youtube.com/JPAModeler>

**Comment by hmbrito** [comment]
Thank You for your attention e congratulations, JPA Modeler is fantastic. 

    Em Terça-feira, 24 de Janeiro de 2017 10:50, Gaurav Gupta <notifications@github.com> escreveu:
 

 This bug is already detected and closed in local branch , Fix will be
available in next release v3.1 .

Thanks and kind regards,
Gaurav Gupta
<https://twitter.com/intent/follow?original_referer=https%3A%2F%2Fabout.twitter.com%2Fresources%2Fbuttons&ref_src=twsrc%5Etfw&region=follow_link&screen_name=jpamodeler&tw_p=followbutton>
<http://jpamodeler.github.io/> <https://www.youtube.com/JPAModeler>
